sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -447,6 +454,17 @@ public static void startFromLibrary(Context context, Class<? extends AppCenterSe
         return getInstance().setInstanceMaxStorageSizeAsync(storageSizeInBytes);
     }
 
+    /**
+     * Set an authentication provider to refresh authentication tokens when they are about to expire or are already expired.
+     *
+     * @param authProvider authentication provider defined by the application or null to remove it.
+     */
+    @SuppressWarnings("WeakerAccess")
+    // TODO Remove when used in the app without reflection (after release).
```

Comment:
This TODO is for `SuppressWarnings`, please move it above `SuppressWarnings`. I understood it to remove the API after release.